web/main.rb

```diff
@@ -4,31 +4,38 @@
 require 'string-utility'
 require_relative '../utils/utilities'
 
-gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
-demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
-adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
+$gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
+$demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
+$adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
```

Comment:
That's strange, but the best alternative here would probably be to do one of two things
* Enclose them in a module and access them from that
* or pass them as arguments to the analyze method
I would go with option 2, it also makes analyze more generic, which is nice in and of itself